Services/COPage/classes/class.ilPCParagraphGUI.php

```diff
@@ -652,7 +652,7 @@ public static function getCharStyleSelector(
         //$selection->setSelectedValue($a_selected);
         $selection->setUseImages(false);
         $selection->setOnClickMode(ilAdvancedSelectionListGUI::ON_ITEM_CLICK_NOP);
-        if (is_string($a_use_callback)) {
+        if (is_string($a_use_callback)) {// @TODO: PHP8 Review: Thje
```

Comment:
??

Reply:
The `is_string` check does not make sense, because `$a_use_callback` it is **always** a `string`.